codebase/billing-service-secondary-async/src/main/resources/application.conf

```diff
@@ -0,0 +1,84 @@
+akka.stream.alpakka.s3 {
```

Comment:
I know it's generally over simplification (in real application it would be as you did) but wouldn't it be simpler if all of these was in a single file (just perhaps in different sections)?

Reply:
done